spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/ZuulProperties.java

```diff
@@ -838,4 +852,23 @@ public String toString() {
 				.append("}").toString();
 	}
 
+	public class Enabled {
+		private boolean enabled = false;
```

Comment:
Why this class?

Reply:
Hi Spencer,

I could easily just put the `enabled` flag within the local `TraceProxyRequestHelper` property class, but I was trying to suggest and open an opportunity for other zuul properties to move more towards the spring-boot-actuator convention where enabling/disabling properties looks like `prefix.<some-property>.enabled` vs. `zuul.<some-property>-enabled`.  I figured that other properties could simply make use of the `Enabled` class, if desired.

I could also just make my property `zuul.trace-proxy-request-helper-enabled`, too, to be more consistent with all the other zuul properties, but was following @ryanjbaxter's property name suggestion.

Please let me know which way you'd prefer to go and I'll be happy to update my PR.